src/pages/SSLModule/list/index.tsx

```diff
@@ -46,11 +53,13 @@ const List: React.FC = () => {
     // TODO: need to check api response
     {
       title: '关联路由',
-      dataIndex: [],
+      dataIndex: 'relatedRouting',
+      hideInSearch: true,
     },
     {
       title: '过期时间',
-      dataIndex: [],
+      dataIndex: 'expireTime',
```

Comment:
🤔Please use `expiredAt` or `expiredTime` or `notAfter`, should have a consensus with APISIX.